packages/assert/src/assert.ts

```diff
@@ -16,19 +16,30 @@ import { A11yConfig, AxeResults, getViolations } from '@sa11y/common';
  */
 export type A11yCheckableContext = Document | Node | string;
 
+export async function getA11yResults(
```

Comment:
For the sake of better readability, should we rather add this logic inside  'getViolationsJSDOM' and call it 'getA11yResults' may be?